packages/hash/api/src/graph/workspace-types.ts

```diff
@@ -22,12 +26,22 @@ export let WORKSPACE_TYPES: {
     orgName: PropertyTypeModel;
     orgSize: PropertyTypeModel;
     orgProvidedInfo: PropertyTypeModel;
+
+    // OrgMembership-related
+    responsibility: PropertyTypeModel;
   };
   entityType: {
     user: EntityTypeModel;
     org: EntityTypeModel;
+    orgMembership: EntityTypeModel;
+  };
+  linkType: {
+    // User-related
+    hasMembership: LinkTypeModel;
+
+    // OrgMembership-related
+    ofOrg: LinkTypeModel;
```

Comment:
Are these the same links in opposite directions? If yes, why do we need this? I thought we would try to avoid this.

Reply:
No, they are the link either side of the membership entity

`User -- hasMembership --> Membership -- ofOrg --> Org`

This is the pattern we're testing out to associate properties to a relationship (in this case, membership of an org)

Reply:
I see, thanks for clarifying 🙂